### PR TITLE
Make templates translatable

### PR DESCRIPTION
**Pull request by timobrembeck**
This is not necessarily required for #125 (since I don't use the admin views of this app and only had the translation of the model fields/properties in mind), but for consistency reasons, I think this addition makes sense.

### CODE REVIEW

File: linkcheck/views.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 from django.shortcuts import render
 from django.templatetags.static import static
 from django.urls import reverse, NoReverseMatch
+from django.utils.translation import gettext_lazy as _
```
Comment (claudep) [inline linkcheck/views.py]: I don't think the lazy version is needed in this file.

File: linkcheck/templates/linkcheck/paginator.html [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
+            <a href="?{{qry_data}}&page={{ pages.previous_page_number }}">&lt;&lt; {% trans "Previous" %}</a>
         {% else %}
-            &lt;&lt; Previous
+            &lt;&lt; {% trans "Previous" %}
         {% endif %}
 
-        <span class="current">
-            Page {{ pages.number }} of {{ pages.paginator.num_pages }}.
+        <span class="current" style="margin: 0 25px;">
+            {% trans "Page" %} {{ pages.number }} {% trans "of" %} {{ pages.paginator.num_pages }}
```
Comment (claudep) [inline linkcheck/templates/linkcheck/paginator.html]: This is no good practice, segmenting sentences into separate translatable words. Use `{% blocktranslate %}` with variables instead.

File: linkcheck/templates/linkcheck/coverage.html [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
+            <th>{% trans "Model" %}</th>
+            <th>{% trans "Covered" %}</th>
+            <th>{% trans "Suggested config" %}</th>
         </tr>
         {% for model in coverage_data %}
             <tr>
                 <td>{{ model.name }}</td>
                 <td style="font-weight: bold;color:{% if model.is_covered %}green{% else %}red{% endif %};">
-                    {{ model.is_covered }}
+                    {% if model.is_covered %}{% trans "Yes" %}{% else %}{% trans "No" %}{% endif %}
```
Comment (claudep) [inline linkcheck/templates/linkcheck/coverage.html]: we could use the `yesno` filter here.

File: linkcheck/templates/linkcheck/base_linkcheck.html [hunk 4]
```diff
@@ -1,8 +1,20 @@
 {% extends "admin/change_list.html" %}
-{% block breadcrumbs %}<div class="breadcrumbs"><a href="/admin/">Home</a>&nbsp;&rsaquo;&nbsp;Link Checker</div>{% endblock %}
+{% load i18n %}
+
+{% block title %}
+    {% trans "Link Checker" %} {{ block.super }}
```
Comment (claudep) [inline linkcheck/templates/linkcheck/base_linkcheck.html]: Recent Django docs suggests using `translate` instead of `trans`.

**Review by claudep — CHANGES_REQUESTED** [review]
Thanks, some points to address.

File: linkcheck/templates/linkcheck/suggested_configs.html [hunk 5]
```diff
@@ -1,3 +1,4 @@
+{% load i18n %}
```
Comment (claudep) [inline linkcheck/templates/linkcheck/suggested_configs.html]: I don't think the library needs to be imported here.

**Review by claudep — APPROVED** [review]
Apart from the unneeded import, this looks good, thanks!

File: linkcheck/templates/linkcheck/suggested_configs.html (hunk 5 above)
Comment (timobrembeck) [reply linkcheck/templates/linkcheck/suggested_configs.html]: You're right, I removed it.

**Pull request merged**